stable/gce-ingress/templates/rbac.yaml

```diff
@@ -0,0 +1,34 @@
+{{ if .Values.rbac.enabled -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "gce-ingress.fullname" . }}
```

Comment:
see https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md - use a helper for the serviceaccount name

Reply:
wicked thanks!